lib/honeybee/construction_set.rb

```diff
@@ -34,7 +34,7 @@
 module Honeybee
   class ConstructionSetAbridged < ModelObject
 
-    def defaults
+    def self.defaults
```

Comment:
I am not sure why you are adding this here as none of the other classes have it. Is this just a Ruby styling thing?

Reply:
I was seeing an error when I am trying to use the self.defaults method inside a from_openstudio self.(method). Self in a ruby method essentially means it is a class method. I am not using this in the construction set class so I can remove it.